lib/kitchen/directions/bake_numbered_table/v1.rb

```diff
@@ -22,14 +22,28 @@ def bake(table:, number:)
       table.parent.add_class('os-column-header-container') if table.column_header?
 
       # TODO: extra spaces added here to match legacy implementation, but probably not meaningful?
-      new_summary = "#{table_label}  "
+      new_summary = "#{table_label} "
       new_caption = ''
+      caption_title = ''
+
+      if (title = table.first("span[data-type='title']")&.cut)
```

Comment:
I think it is time for a `titles` method because we do this search a lot.  After this PR, can you make another kitchen PR that adds `ElementEnumeratorBase#titles` and `ElementBase#titles`.  @KarinaMendez2 just did something similar for metadata searches in https://github.com/openstax/kitchen/pull/133.